tests/views_wpunit/Tribe/Events/Views/V2/HooksTest.php

```diff
@@ -113,4 +115,36 @@ public function test_filter_redirect_canonical_will_not_redirect_embed( array $c
 
 		$this->assertEquals( $expected, $filtered );
 	}
+
+	/**
+	 * Validate the posts in the title are stored from the view repository search.
+	 * This is done via the hooks in the Views/Hooks class.
+	 */
+	public function test_events_for_title_stored() {
+		for ( $i = 1; $i < 20; $i ++ ) {
+			$date = "2020-06-$i 08:00:00";
+			tribe_events()->set_args( [
+				                          'start_date' => $date,
+				                          'timezone'   => 'America/New_York',
+				                          'duration'   => 3 * HOUR_IN_SECONDS,
+				                          'title'      => 'Test Event',
+				                          'status'     => 'publish',
+			                          ] )->create();
+		}
+
+		$title = tribe( Title::class );
+		$this->assertEmpty( $title->get_posts() );
+		$context = tribe_context()->alter( [
+			                                   'single' => false,
+			                                   'event_post_type' => true,
+
+			                                   'event_display' => 'list',
+			                                   'event_date'    => '2020-06-01',
+		                                   ] );
```

Comment:
```suggestion
		$context = tribe_context()->alter( 
            [
            'single' => false,
            'event_post_type' => true,

            'event_display' => 'list',
            'event_date'    => '2020-06-01',
            ] 
        );
```
I'm surprised this isn't triggering the phpcs checks. Same above.

Reply:
Oh, because it's in a test. 😐